modules/adformOpenRTBBidAdapter.js

```diff
@@ -66,11 +66,28 @@ export const spec = {
         };
         if (props) {
           asset.id = props.id;
+          let wmin, hmin, w, h;
+          let aRatios = bidParams.aspect_ratios;
+
+          if (aRatios) {
+            aRatios = aRatios[0];
+            wmin = aRatios.min_width || 0;
+            hmin = aRatios.ratio_height * wmin / aRatios.ratio_width | 0;
```

Comment:
Are there default values for `aRatios.ratio_height` and `aRatios.ratio_width`?  Is it possible that either of these could be 0 or undefined?

Reply:
It's just a fallback to minWidth/minHeight targeting, so it's OK to have `NaN` as an expression result. Bitwise OR operator will convert it to 0.